test/amp-caching-sw.js

```diff
@@ -0,0 +1,10 @@
+function ampAssetsCaching() {
```

Comment:
This is the method in the `amp-caching` module right? Why duplicate into the test?

Reply:
removed this file to be auto generated by the test